src/UICircularProgressRing/UICircularProgressRingLayer.swift

```diff
@@ -411,12 +445,24 @@ class UICircularProgressRingLayer: CAShapeLayer {
         valueLabel.textAlignment = .center
         valueLabel.textColor = fontColor
 
-        valueLabel.update(withValue: value,
-                          valueIndicator: valueIndicator,
-                          rightToLeft: rightToLeft,
-                          showsDecimal: showFloatingPoint,
-                          decimalPlaces: decimalPlaces,
-                          valueDelegate: valueDelegate)
+        if (valueTextAttributes != nil) || (indicatorTextAttributes != nil) {
```

Comment:
If `nil` isn't important why not just call this function outright and remove the old function?